firmware/integration/rusefi_config.txt

```diff
@@ -1397,7 +1398,12 @@ float[CLT_CURVE_SIZE] iacCoasting;    CLT-based idle position for coasting (used
 	linear_sensor_s highPressureFuel;
 	linear_sensor_s lowPressureFuel;
 
-uint8_t[536] unused3328;;"units", 1, 0, -20, 100, 0
+! issue 455: RPM hard limit depending on CLT
+int8_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpmBins;CLT-based target RPM for hard limit depending on CLT LFAstyle;"C",        1,     0,   -70,    120,  0
```

Comment:
what is `LFAstyle`?

Reply:
When this issuer is opens there is a comment "Variable RPM hard limit depending on engine temperate would be a nice feature right from Lexus LFA", so, I called this Lexus LFA style CLT Rev Limiter

Reply:
maybe `...hard limit depending on CLT, just like a Lexus LFA`

Reply:
Ok, anyway I put the [WIP] tag, so will clear the old lines and change this comment on the final release of this ticket. If it possible.

Reply:
yes no problem - you can continue pushing to your branch and it'll just update the PR when you `git push`